src/Blocks/BlockFarmland.h

```diff
@@ -127,6 +127,16 @@ class cBlockFarmlandHandler :
 
 		return false;
 	}
+
+	/** Checks whether the block has an effect on growing the plant */
+	virtual bool CanSustainPlant(BLOCKTYPE a_Plant) override
+	{
+		return ((a_Plant == E_BLOCK_CROPS)
+			|| (a_Plant == E_BLOCK_CARROTS)
+			|| (a_Plant == E_BLOCK_POTATOES)
+			|| (a_Plant == E_BLOCK_MELON_STEM)
+			|| (a_Plant == E_BLOCK_PUMPKIN_STEM));
```

Comment:
We usually use a slightly different style:
```cpp
return (
  (a_Plant == E_BLOCK_CROPS) ||
  (a_Plant == E_BLOCK_CARROTS)
);
```
(Same as in `if` conditions if they're long)